docs/vaahflutter/directory_structure/root/d_lib.md

```diff
@@ -0,0 +1,47 @@
+# Lib
+
+[[toc]]
+
+## What is this directory about
+
+All your app code will reside in the lib folder. Make sure to plan your code structure before starting to develop your app. Otherwise, it can get confusing quickly without proper organization.
+
+This is the most important folder in the project, used to write most of the dart code. By default, the lib folder contains the main.dart file, which is the application’s entry point, in case of multiple environments lib can have multi main files.
+
+In our repo you can see following files and directories in lib folder
+```{20}
+project
+│   
+└─── lib
+    │
+    └─── models
+    |   | ...
+    |
+    └─── views
+    |   | ...
+    |
+    └─── controllers
+    |   | ...
+    |
+    └─── resources
+    |   | ...
+    |
+    └─── routes
+    |   | ...
+    |
+    └─── vaahextendflutter
+    |   | ...
+    |
+    └───  app_config.dart
+    |
+    └───  main.dart
+```
+
+| **Directory name** | **Documentation link** | **Description** |
+| --- | --- | --- |
+| models | [check this](../lib/models) | This directory is project specific and contains models for the project |
+| views | [check this](../lib/views) | This directory is also project specific and contains UI screens for the project |
```

Comment:
@we-prajapati-c001 for the view we are using `Atomic Design Methodology`, explain and link it to https://atomicdesign.bradfrost.com/chapter-2/

Reply:
Okay sir will do.

Reply:
https://youtu.be/N8Y4NuUfl8I